docsrc/docextract_settings.rst

```diff
@@ -298,7 +296,7 @@ Setting::
 
 Return style information for the token. Example Return::
 
-    {"bold": true, "underlined": true, "italics": true, "font_size": 14, "background_color": "hex", "text_color": "hex"}
+    {"bold": true, "underlined": true, "italics": true, "font_size": 14, "background_color": <hex> color value, "text_color": <hex> color value}
```

Comment:
I think this is where Slater suggested "FFFFF" as the default sample, but to me this reads the same

Reply:
I talked this through with him and we agreed to just include <hex>